File: sphinx/java-intro/source/code/src/main/java/com/oneoffcoder/java/clazz/GetterSetter.java
package com.oneoffcoder.java.clazz;

public class GetterSetter {

  public static class Car {
    private String make;
    private String model;
    private int year;

    public Car(String make, String model, int year) {
      this.make = make;
      this.model = model;
      this.year = year;
    }

    public String getMake() {
      return make;
    }

    public String getModel() {
      return model;
    }

    public int getYear() {
      return year;
    }

    public void setMake(String make) {
      this.make = make;
    }

    public void setModel(String model) {
      this.model = model;
    }

    public void setYear(int year) {
      this.year = year;
    }
  }

  public static void main(String[] args) throws Exception {
    var car = new Car("Honda", "Accord", 2019);

    System.out.println(car.getMake() + " " + car.getModel() + " " + car.getYear());
  }

}

File: sphinx/python-intro/source/code/oneoffcoder/clazz/gettersetter.py
class Car(object):
    def __init__(self, make, model, year):
        self.make = make
        self.model = model
        self.year = year

    def get_make(self):
        return self.make

    def get_model(self):
        return self.model

    def get_year(self):
        return self.year

    def set_make(self, make):
        self.make = make

    def set_model(self, model):
        self.model = model

    def set_year(self, year):
        self.year = year


car = Car('Honda', 'Accord', 2019)

car.set_make('Toyota')
car.set_model('Supra')
car.set_year(2019)

print(f'{car.get_make()} {car.get_model()} {car.get_year()}')